src/sdk/models/entityTypes.ts

```diff
@@ -1,6 +1,27 @@
 import { IAuditedEntity, IEntityIdentifier } from './base';
 import { Category } from './categories';
 
+export enum ConfidenceLevel {
+	None = 'none',
+	VeryWeak = 'very_weak',
+	Weak = 'weak',
+	Medium = 'medium',
+	Strong = 'strong',
+	VeryStrong = 'very_strong'
+}
+export interface EntityTypeRegexPattern {
+	description: string;
+	pattern: string;
+	priority: number;
+	confidence: ConfidenceLevel;
+}
+
+export interface EntityTypeKeyword {
```

Comment:
I have a branch with this and support for bringing in the actual keywords - https://github.com/portalarchitects/skysync-cli/commit/689f00a927c9be30f085dea2f94230f0fba5a7ad. Not a PR just yet but close. So two options:
1) Could you take out EntityTypeKeyword and keywords property below in EntityType and we will use the ones in my PR?
or 
2) We can merge what I need into this. But includes some additional files.

Reply:
Thanks Jim. I went ahead and removed it.